### ISSUE TITLE
Whats wrong with ref?

### ISSUE THREAD

**Issue by IftekherSunny** [atlassian/react-beautiful-dnd#924]
Hi, 

Getting ref error. I Can't figure it out why.

Here the screenshot from chrome devtools [Screenshot](https://monosnap.com/direct/8zXP7cGDS5yQ69fi443OZAvtKCIXnA)

**Error:**

```
react-beautiful-dnd.esm.js:5000 raw Error: Invariant failed: 
    provided.innerRef has not been provided with a HTMLElement.

    You can find a guide on using the innerRef callback functions at:
    https://github.com/atlassian/react-beautiful-dnd/blob/master/docs/guides/using-inner-ref.md
  
  
    at invariant (tiny-invariant.esm.js:11)
    at throwIfRefIsInvalid (react-beautiful-dnd.esm.js:5583)
    at Droppable._this.setRef (react-beautiful-dnd.esm.js:5617)
    at commitAttachRef (react-dom.development.js:14804)
    at commitAllLifeCycles (react-dom.development.js:15910)
    at HTMLUnknownElement.callCallback (react-dom.development.js:145)
    at Object.invokeGuardedCallbackDev (react-dom.development.js:195)
    at invokeGuardedCallback (react-dom.development.js:248)
    at commitRoot (react-dom.development.js:16075)
    at completeRoot (react-dom.development.js:17463)
```

**Code**

```js
// Droppable
    <Droppable droppableId={path.join("-")}>
        {(provider) => (
          <Accordion
            innerRef={provider.innerRef}
            provider={provider}
            className="qx-accordion"
          >
           <h2>hi</h2>
          </Accordion>
        )}
      </Droppable>
```

```js
  // accordion
export class Accordion extends Component {
  render() {
    const classes = classnames('accordion', this.props.className);

    return (
      <div className={classes} {...this.props.provider.droppableProps} id={this.props.className} ref={this.props.innerRef}>
        {this.props.children}
      </div>
    )
  }
}
```

**Comment by balonsom** [comment]
Hi @IftekherSunny if you are using styled-components v4 the innerRef prop has been removed, you can check it out [here](https://www.styled-components.com/docs/api#innerref-prop).

> NOTE
> The "innerRef" prop was removed in styled-components v4 in favor of the [React 16 forwardRef API](https://reactjs.org/docs/forwarding-refs.html). Just use the normal ref prop instead.

To solve it just change `innerRef=` with `ref=`:
```
...
{(provider)} => (
    <Accordion
        ref={provider.innerRef}
        provider={provider}
        className="qx-accordion"
    >
        <h2>hi</h2>
    </Accordion>
)
...
```

**Comment by alexreardon** [comment]
@balonsom could you add this as a note to our `innerRef` guide? This keeps tripping people up

**Comment by balonsom** [comment]
@alexreardon done!

**Comment by IftekherSunny** [comment]
@balonsom isn't working

```
Uncaught Error: Invariant failed: 
    provided.innerRef has not been provided with a HTMLElement.

    You can find a guide on using the innerRef callback functions at:
    https://github.com/atlassian/react-beautiful-dnd/blob/master/docs/guides/using-inner-ref.md
  
  
    at invariant (tiny-invariant.esm.js:11)
    at throwIfRefIsInvalid (react-beautiful-dnd.esm.js:5583)
    at Droppable._this.setRef (react-beautiful-dnd.esm.js:5617)
    at commitAttachRef (react-dom.development.js:14804)
    at commitAllLifeCycles (react-dom.development.js:15910)
    at HTMLUnknownElement.callCallback (react-dom.development.js:145)
    at Object.invokeGuardedCallbackDev (react-dom.development.js:195)
    at invokeGuardedCallback (react-dom.development.js:248)
    at commitRoot (react-dom.development.js:16075)
    at completeRoot (react-dom.development.js:17463)
```

**Comment by IftekherSunny** [comment]
@balonsom from chrome tools http://take.ms/QGR4d

**Comment by alexreardon** [comment]
My understanding is that this has been closed by #926
